### PR TITLE
Fix instrument default view

### LINKED ISSUE #23486: Instrument view error
Tab `Render` default should be `Full 3D` but is `CylindricalY`

Run `LoadEmptyInstrument` for FIGARO and open the instrument view:
I see the following:

errormessage

And the default view is not full 3D:
cylindricaly

Please note that FIGARO is a rectangular detector with special dimensions

### PR DESCRIPTION
**Pull request by vreimund**
The instrument default view got broken https://github.com/mantidproject/mantid/commit/c2a4655e6cf3a34f0e8693be0ab9961f6d5f9025?diff=unified.

**To test:**

Read the documentation about the instrument definition file -> default-view meaning
Should be:
- 3D is default except OpenGL is explicitly switched off

On master:
Enable OpenGL: View->Preferences->Mantid->Options->UseOpenGL in Instrument View
Try `LoadEmptyInstrument` for an instrument and you will not get the Full 3D view.

Fixes #23486.

No release notes since not yet released.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by LamarMoore** [comment]
@reimundILL Don't you need to update the IDF file as part of this PR or isn't it stored in the `instruments` folder?

File: qt/widgets/instrumentview/src/InstrumentWidget.cpp [hunk 1]
```diff
@@ -264,9 +264,9 @@ void InstrumentWidget::init(bool resetGeometry, bool autoscaling,
       QString defaultView =
           QString::fromStdString(m_instrumentActor->getDefaultView());
       if (defaultView == "3D" &&
-          Mantid::Kernel::ConfigService::Instance()
-              .getValue<bool>("MantidOptions.InstrumentView.UseOpenGL")
-              .get_value_or(false)) {
+          !Mantid::Kernel::ConfigService::Instance()
```
Comment (LamarMoore) [inline qt/widgets/instrumentview/src/InstrumentWidget.cpp]: Why this change? Isn't it essentially the same thing? Were you getting errors with the stored value not evaluation to false?

**Review by LamarMoore — CHANGES_REQUESTED** [review]

**Comment by reimundILL** [comment]
This issue is general. I think even if we specify the default view in the IDF, it depends on the openGL setting if enabled finally.

I keep in mind to open a future issue for our IDFs if the users wish a change. The default 3D seems to be ok at the moment and if a user switches off the openGL, the warning may be the correct answer.

**Review by LamarMoore — APPROVED** [review]

**Comment by gvardany** [comment]
Just to clarify, if an IDF does not explicitly specify the view, it has to be assumed as 3D. This is no longer the case. There should be no need in editing all the IDFs to add the setting, but rather the logic should be fixed, which got broken after that commit.

**Pull request merged**